webextension/css/lt-marker-style.css

```diff
@@ -0,0 +1,98 @@
+.lt-marker-container {
+  font-size: 13px;
+  font-family: sans-serif;
+}
+
+.lt-buttons {
+  box-shadow: 0px 5px 11px -2px rgba(0, 0, 0, 0.18),
+    0px 4px 12px -7px rgba(0, 0, 0, 0.15);
+  border-radius: 50%;
+  display: block;
+  width: 16px;
+  height: 16px;
+  -webkit-transition: all 0.2s ease-out;
+  transition: all 0.2s ease-out;
+  cursor: pointer;
+  z-index: 1000;
+}
+
+.lt-buttons:active,
+.lt-buttons:focus,
+.lt-buttons:hover {
+  box-shadow: 0 0 4px rgba(0, 0, 0, 0.14), 0 4px 8px rgba(0, 0, 0, 0.28);
+}
+
+.lt-remind-btn {
+  background-color: #fff;
+  opacity: 0.5;
+  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4QkSCgkBTZ6R6wAAAExJREFUOMtjFEl785+BAsDEQCEYNWDUANoY8HqmMEl8JmySxNIMDAwMDCJpb/6LpL35DwPIbEJ8kbQ3/1FcIJr+liSagYGBgZHS7AwA08JR4ozGxZoAAAAASUVORK5CYII=");
```

Comment:
yes, we should do this way because we have to support FF and Chrome. Last time, I have used this one but It didn't work on FF.

```      body {
        background-image:url('chrome-extension://__MSG_@@extension_id__/background.png');
       }```